.claude/skills/proxmox-infrastructure/tools/check_ceph_health.py

```
@@ -0,0 +1,458 @@
+        self.health = CEPHHealth(
+            status="UNKNOWN",
+            num_osds=0,
+            num_up_osds=0,
+            num_in_osds=0,
+            num_pgs=0,
+            num_active_clean_pgs=0
+        )
+
+    def _validate_node(self, node: str) -> bool:
+        """Validate node is a valid hostname or IP address"""
+        import re
```

Ruff (0.14.1)

Redefinition of unused `re` from line 28

Remove definition: `re`

(F811)